examples/sklearn_model/README.rst

```diff
@@ -14,6 +14,9 @@ Then, run either `python start_service.py` to run
 flask service, or `python model_create_image.py` to create
 and run docker container.
 
-After that, you can run `python client.py %some_number%`
+After that, you can run `python client.py %some_number% %some_other_number%`
```

Comment:
`smart_client.py`? do we need both clients?

Reply:
According to what was discussed orally a couple of months ago we need. `HTTPClient` is a Pythonic "battery included" and provides data scientists with a familiar interface. But we have to explicitly demonstrate that Ebonite services are regular HTTP services and could be easily consumed via Python commodity libraries (e.g., `requests`) or even non-Python platforms (e.g., Java).